test/rules/use_setters_to_change_properties.dart

```diff
@@ -8,13 +8,15 @@ abstract class A {
   int _w;
   int _x;
 
-  void setW(int w) => this._w = w; // LINT
```

Comment:
Should we have both tests (with and without `this`) to ensure that we recognize `_w` in either context?

Reply:
Not a bad idea.  Done!